lib/src/cupertino_page_route.dart

```diff
@@ -636,7 +636,7 @@ class _CupertinoBackGestureController<T> {
     @required this.controller,
   }) : assert(navigator != null),
        assert(controller != null) {
-    navigator.didStartUserGesture();
+    //navigator.didStartUserGesture();
```

Comment:
Can you clarify which use-case do you have with keyboard?

Reply:
@PiN73 If there are two pages with autofocused text in each. If the user navigates back from one page to another by swiping back, the field will lose autofocus

Reply:
I think we shouldn't disable these callbacks as it can break other things. If you want to allow focusing during user gesture, I think it would be better to patch [_shouldIgnoreFocusRequest in widgets/routes.dart](https://github.com/flutter/flutter/blob/9b2d32b605630f28625709ebd9d78ab3016b2bf6/packages/flutter/lib/src/widgets/routes.dart#L758) making it always return false whatever the value of userGestureInProgress is

Reply:
As widgets/routes.dart isn't a part of this package, I'm going to keep the original behavior

Reply:
> I think we shouldn't disable these callbacks as it can break other things. If you want to allow focusing during user gesture, I think it would be better to patch [_shouldIgnoreFocusRequest in widgets/routes.dart](https://github.com/flutter/flutter/blob/9b2d32b605630f28625709ebd9d78ab3016b2bf6/packages/flutter/lib/src/widgets/routes.dart#L758) making it always return false whatever the value of userGestureInProgress is

What things it could break?